src/modules/avformat/filter_avfilter.c

```diff
@@ -150,6 +156,35 @@ static void set_avfilter_options( mlt_filter filter, double scale)
 	}
 }
 
+static void send_avformat_commands(mlt_filter filter, mlt_frame frame, private_data* pdata)
+{
+#if LIBAVUTIL_VERSION_INT >= ((56<<16)+(35<<8)+101)
+	mlt_properties prop = MLT_FILTER_PROPERTIES(filter);
+	mlt_position position = mlt_filter_get_position( filter, frame );
+	int length = mlt_filter_get_length2( filter, frame );
+	int count = mlt_properties_count(prop);
+	int i;
+	for (i = 0; i < count; i++)
+	{
+		char *name = mlt_properties_get_name(prop, i);
+		if (!strncmp(name, PARAM_PREFIX, PARAM_PREFIX_LEN))
+		{
+			const AVOption *opt = av_opt_find( pdata->avfilter_ctx->priv, name + PARAM_PREFIX_LEN, 0, 0, 0 );
+			if (opt && opt->flags & AV_OPT_FLAG_RUNTIME_PARAM) {
+				mlt_properties_set_double(prop, "_avfilter_temp", mlt_properties_anim_get_double(prop, name, position, length));
```

Comment:
Should we be applying resolution scaling to the value before sending the command here?

Reply:
yes, good find

Reply:
done

Reply:
done